packages/material-ui-lab/src/Skeleton/Skeleton.js

```diff
@@ -1,13 +1,14 @@
 import * as React from 'react';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
-import { withStyles } from '@material-ui/core/styles';
+import { fade, withStyles } from '@material-ui/core/styles';
 
 export const styles = (theme) => ({
   /* Styles applied to the root element. */
   root: {
     display: 'block',
-    backgroundColor: theme.palette.action.hover,
+    // create a "on paper" color with sufficient contrast
```

Comment:
nit: do you prefer to start comments with lowercase but reserve capital for more like "official" comments that end up in docs (e.g. other comments in this file)?

Reply:
I'm lazy and rely on review to clean these up :wink: